README.md

```diff
@@ -11,12 +13,17 @@
 ### Exercise (basic)
 
 Implement the fizz buzz list. If x is the i'th element of `fizzBuzz` then
-  - if i is (only) divisible by 3 then x is the string `"Fizz"`; 
+  - if i is (only) divisible by 3 then x is the string `"Fizz"`;
   - if i is (only) divisible by 5 then x is the string `"Buzz"`;
   - if i is divisible by 3 and 5 then x is the string `"FizzBuzz"`;
   - otherwise x is the string `""`;
 `fizzBuzz` should be an infinite list. You may define it any way you please.
 
+### Exercise (intermediate)
+
+Write a function that implements an aproximation to sqrt for integers.
```

Comment:
I'm assuming you mean `floor . sqrt`, in essence. Would be worth indicating that (examples?), without giving the types away (my assumption here is the function has type `Integer -> Maybe Integer`).

Reply:
So my solution was:
```haskell
sqrtInt = \n -> last [x | x <- [1..n], x*x <= n]
```
I didn't know about `floor`.  I'll think about how to phrase the question  to basically say "don't just do `floor . sqrt`"